src/viv_toml_config.c

```diff
@@ -92,6 +92,13 @@ static struct string_map_pair scroll_method_map[] = {
     NULL_STRING_MAP_PAIR,
 };
 
+static struct string_map_pair click_method_map[] = {
+    {"none", LIBINPUT_CONFIG_CLICK_METHOD_NONE},
+    {"area", LIBINPUT_CONFIG_CLICK_METHOD_BUTTON_AREAS},
```

Comment:
The tests have picked up a typo - looks like this should be `"areas"`?

Reply:
excellent, fixed